jp/components/grid_editing.md

```diff
@@ -91,7 +91,7 @@ this.grid.addRow(record);
 
 #### グリッド データの更新
 
-Updating data in the grid is achieved through `updateRow` and `updateCell` methods but **only if primary key for the grid is defined**. You can also directly update a cell and row value through their `update` method.
+グリッド データの更新は、**グリッドでプライマリキーが定義されている場合のみ** `updateRow` と `updateCell` メソッドで行うことができます。セルおよび行の値を直接更新するには `update` メソッドを使用できます。
```

Comment:
I cannot find a way to add a comment where I want to!  Line 113 was added and needs to be localized as well!